src/services/navigationBar.ts

```diff
@@ -681,6 +682,9 @@ namespace ts.NavigationBar {
                 return isExternalModule(sourceFile)
                     ? `"${escapeString(getBaseFileName(removeFileExtension(normalizePath(sourceFile.fileName))))}"`
                     : "<global>";
+            case SyntaxKind.ExportAssignment:

```

Comment:
I think it would be better to have `export=` as the name when using `export = abc` statement and `default` in the export default assignments?

Reply:
This is in - thanks